registry.k8s.io/images/k8s-staging-cluster-api-helm/images.yaml

```diff
@@ -24,3 +24,4 @@
     "sha256:c04d0408ee13a5bb18c27db6ded9890bef53b2701b834a6d3ba9dbdcefc51769": ["v0.4.2"]
     "sha256:7364b2462b0d820a59f72c0298b6776bc536758f9501c3c77d500d6031621d5f": ["v0.5.0"]
     "sha256:60d7afc1745efd478cfa145b36540d2bf5d0cbb89491b0924e3cf4450f368286": ["v0.5.1"]
+    "sha256:db44c06875f1af832f5f192ea46424385a51e643454caf84501feb953f2a4d52": ["v0.5.2"]
```

Comment:
SHA `db44c06875f1` matches cluster-api-helm-controller v0.5.2 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-helm/global/cluster-api-helm-controller@sha256:db44c06875f1af832f5f192ea46424385a51e643454caf84501feb953f2a4d52/details?project=k8s-staging-cluster-api-helm).